cypress/support/generate.js

```diff
@@ -0,0 +1,12 @@
+import { faker } from '@faker-js/faker';
+
+function generateUser() {
+  const randomNumber = Math.random().toString().slice(2, 6);
+  const username = faker.internet.userName() + '_' + randomNumber;
+  const email = `${username}@mail.com`;
+  const password = '12345Qwert!';
```

Comment:
Hardcoding passwords in your code can be a security risk, even for test data. Consider generating a random password for each user instead. This could also help you catch potential issues with different password inputs.